src/background.js

```diff
@@ -21,15 +21,26 @@ function init() {
   browser.alarms.onAlarm.addListener(handleWake);
   browser.notifications.onClicked.addListener(handleNotificationClick);
   browser.runtime.onMessage.addListener(handleMessage);
-  handleWake();
+  browser.storage.local.get().then(items => {
```

Comment:
nit: may want to move this big block into its own function, if only to keep that `init()` function small & clean. But, not a review-stopper.